rfc/0023-kubewarden-stack-telemetry.md

```diff
@@ -0,0 +1,338 @@
+|              |                                  |
+| :----------- | :------------------------------- |
+| Feature Name | Kubewarden environment telemetry |
+| Start Date   | 2025-08-28                       |
+| Category     | telemetry                        |
+| RFC PR       | [fill this in after opening PR]  |
```

Comment:
This can be filled now

Reply:
Reopening, we can set the state to approved.